lib/reporters.js

```diff
@@ -51,10 +51,7 @@ function DataDogReporter(apiKey, appKey, apiHost) {
 }
 
 DataDogReporter.prototype.report = function(series, onSuccess, onError) {
-    if (debug.enabled) {
-        // Only call stringify when debugging.
-        debug('Calling report with %s', JSON.stringify(series));
-    }
+    logDebug('Calling report with %j', series);
```

Comment:
I noticed when changing this to `logDebug` that debug has a `%j` formatter for logging an object as JSON. Not sure if it’s new or if we just didn’t know about it before, but it makes this a little nicer and means we don’t need to wrap this in an `if` statement.